### YamlMime:ManagedReference
items:
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException
  id: WorkflowTerminatedException
  children:
  - System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor
  - System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor(System.String)
  - System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: WorkflowTerminatedException
  nameWithType: WorkflowTerminatedException
  fullName: System.Workflow.ComponentModel.WorkflowTerminatedException
  type: Class
  summary: "Представляет <xref:System.Exception>, возникает при завершении рабочего процесса.</xref:System.Exception> Этот класс не наследуется."
  remarks: >-
    > [!IMPORTANT]

    >  [!INCLUDE[DeprecatedContent](~/add/includes/ajax-current-ext-md.md)]
  syntax:
    content: >-
      [System.Obsolete("The System.Workflow.* types are deprecated.  Instead, please use the new types from System.Activities.*")]

      public sealed class WorkflowTerminatedException : Exception
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor
  id: '#ctor'
  parent: System.Workflow.ComponentModel.WorkflowTerminatedException
  langs:
  - csharp
  name: WorkflowTerminatedException()
  nameWithType: WorkflowTerminatedException.WorkflowTerminatedException()
  fullName: System.Workflow.ComponentModel.WorkflowTerminatedException.WorkflowTerminatedException()
  type: Constructor
  assemblies:
  - System.Workflow.ComponentModel
  namespace: System.Workflow.ComponentModel
  summary: "Инициализирует новый экземпляр <xref href=&quot;System.Workflow.ComponentModel.WorkflowTerminatedException&quot;> </xref> класса."
  syntax:
    content: public WorkflowTerminatedException ();
    parameters: []
  overload: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Workflow.ComponentModel.WorkflowTerminatedException
  langs:
  - csharp
  name: WorkflowTerminatedException(String)
  nameWithType: WorkflowTerminatedException.WorkflowTerminatedException(String)
  fullName: System.Workflow.ComponentModel.WorkflowTerminatedException.WorkflowTerminatedException(String)
  type: Constructor
  assemblies:
  - System.Workflow.ComponentModel
  namespace: System.Workflow.ComponentModel
  summary: "Инициализирует новый экземпляр <xref href=&quot;System.Workflow.ComponentModel.WorkflowTerminatedException&quot;> </xref> класса с указанным сообщением об ошибке."
  syntax:
    content: public WorkflowTerminatedException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Сообщение, описывающее ошибку."
  overload: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Workflow.ComponentModel.WorkflowTerminatedException
  langs:
  - csharp
  name: WorkflowTerminatedException(String,Exception)
  nameWithType: WorkflowTerminatedException.WorkflowTerminatedException(String,Exception)
  fullName: System.Workflow.ComponentModel.WorkflowTerminatedException.WorkflowTerminatedException(String,Exception)
  type: Constructor
  assemblies:
  - System.Workflow.ComponentModel
  namespace: System.Workflow.ComponentModel
  summary: "Инициализирует новый экземпляр <xref href=&quot;System.Workflow.ComponentModel.WorkflowTerminatedException&quot;> </xref> класса с указанным сообщением об ошибке и ссылкой на внутреннее <xref:System.Exception>, являющееся причиной <xref:System.Exception>.</xref:System.Exception> </xref:System.Exception>"
  syntax:
    content: public WorkflowTerminatedException (string message, Exception exception);
    parameters:
    - id: message
      type: System.String
      description: "Сообщение, описывающее ошибку."
    - id: exception
      type: System.Exception
      description: "Внутренний <xref:System.Exception>ссылку.</xref:System.Exception>"
  overload: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor
  parent: System.Workflow.ComponentModel.WorkflowTerminatedException
  isExternal: false
  name: WorkflowTerminatedException()
  nameWithType: WorkflowTerminatedException.WorkflowTerminatedException()
  fullName: System.Workflow.ComponentModel.WorkflowTerminatedException.WorkflowTerminatedException()
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor(System.String)
  parent: System.Workflow.ComponentModel.WorkflowTerminatedException
  isExternal: false
  name: WorkflowTerminatedException(String)
  nameWithType: WorkflowTerminatedException.WorkflowTerminatedException(String)
  fullName: System.Workflow.ComponentModel.WorkflowTerminatedException.WorkflowTerminatedException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor(System.String,System.Exception)
  parent: System.Workflow.ComponentModel.WorkflowTerminatedException
  isExternal: false
  name: WorkflowTerminatedException(String,Exception)
  nameWithType: WorkflowTerminatedException.WorkflowTerminatedException(String,Exception)
  fullName: System.Workflow.ComponentModel.WorkflowTerminatedException.WorkflowTerminatedException(String,Exception)
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor*
  parent: System.Workflow.ComponentModel.WorkflowTerminatedException
  isExternal: false
  name: WorkflowTerminatedException
  nameWithType: WorkflowTerminatedException.WorkflowTerminatedException
